src/pages/[sourceFileId].tsx

```diff
@@ -0,0 +1,73 @@
+import App from '../App';
+import { GetServerSideProps } from 'next';
+import { gQuery } from '../utils';
+import {
+  GetSourceFileSsrDocument,
+  GetSourceFileSsrQuery,
+} from '../graphql/GetSourceFileSSR.generated';
+import { AppHead } from '../AppHead';
+import { registryLinks } from '../registryLinks';
+
+interface SourceFileIdPageProps {
+  data: GetSourceFileSsrQuery['getSourceFile'];
+  id: string;
+}
+
+const SourceFileIdPage = (props: SourceFileIdPageProps) => {
+  return (
+    <>
+      <AppHead
+        title={
+          props.data?.name
+            ? `${props.data?.name} | XState Visualiser`
```

Comment:
This is passed to `og:title` "as-is". I'm not entirely sure about this but I think it's usually preferred to keep the `og:title` succinct - without branding etc. It's one of the reasons why `og:title` exists - `<title/>` was already crammed with things like this and the people behind OpenGraph wanted "cleaner" titles

Reply:
Gotcha, so we should have a different prop for og:title that's cleaner